index.html

```diff
@@ -4,9 +4,50 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <meta name="description" content="This is a portfolio website for Munish Halai">
+    <title>Munish Halai's Portfolio</title>
+    <!-- Links -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
+
+    <!-- Website main section starts here -->
+    <main>
+
+        <!-- Navbar Starts here -->
+
+        <header class="header_navbar">
+            <nav class="navbar">
+                <a href="index.html" class="logo">My Logo</a>
+                <img src="images/nav.png" alt="Navbar" class="nav_icon">
+            </nav>
+        </header>
+        
+        <!-- Navbar Ends here -->
+
+        <!-- Header Banner Starts Here -->
+
+        <section class="banner">
+            <div class="banner_content_wrapper">
+                <h1 class="banner_content">I’m Munish. <br>Glad to see you!</h1>
+                <p class="banner_content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <p class="lets_connect">LET’S CONNECT</p>
+                <div class="banner_social_icons_wrapper">
+                    <a href="index.html"><img class="banner_social_icons" src="images/twitter-icon.svg" alt="Twitter"></a>
+                    <a href="index.html"><img class="banner_social_icons" src="images/linkedin-icon.svg" alt="Linkedin"></a>
+                    <a href="index.html"><img class="banner_social_icons" src="images/social-icon-3.svg" alt="Social"></a>
+                    <a href="index.html"><img class="banner_social_icons" src="images/social-icon-4.svg" alt="Social"></a>
+                    <a href="index.html"><img class="banner_social_icons" src="images/social-icon-5.svg" alt="Social"></a>
+                </div>
```

Comment:
- [ ] It would be better to use `<ul>` and `<li>` tags to group and list your social links as they are important for structuring and organizing content on a web page, improving readability and accessibility, and adding semantic meaning to the content.